.travis.yml

```diff
@@ -19,10 +19,10 @@ env:
   # See https://github.com/bazelbuild/rules_scala/pull/622
   # we want to test the last release
   #- V=0.16.1 TEST_SCRIPT=test_lint.sh
-  - V=0.17.1 TEST_SCRIPT=test_rules_scala.sh
+  #- V=0.17.1 TEST_SCRIPT=test_rules_scala.sh
```

Comment:
Let's uncomment or increase this to the lowest version we support. If we are bumping that, let's update the README.